examples/truffle/contracts/RustCoin.sol

```diff
@@ -1,14 +1,13 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.6.0;
 
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
 
-contract RustCoin is ERC20, ERC20Detailed {
-  constructor() ERC20Detailed("Rust Coin", "RUST", 18) public {
+contract RustCoin is ERC20 {
+  constructor() ERC20("Rust Coin", "RUST") public {
     _mint(msg.sender, 1337 * (10 ** uint256(decimals())));
   }
 
-  function() external payable {
+  receive() external payable {
```

Comment:
This must also be some sort of fancy new feature.

Reply:
`receive` is a payable fallback function. So before you had:
```
function() external {
    // non-payable fallback function
}
function() external payable {
    // payable fallback function
}
```
and now you have
```
fallback() external {
    // non-payable fallback function
}
receive() external payable {
    // payable fallback function
}
```